### DISCUSSION TITLE
SQL Lab Trino, no cost estimate button

### DISCUSSION THREAD

**Discussion by tarekabouzeid** [apache/superset#22603] [Q&A / Help] [answered]
Hi,

I am using Superset 2.0.1 and i have added a Trino connection and enabled "Enable query cost estimation". But when I open data lab I don't see the cost estimate button. It is written inside an info box that query cost estimation " For Presto and Postgres, shows a button to compute cost before running a query." Does that mean this feature is not available for Trino at all ?

Thanks

**Comment by vinhhap** [comment, marked as answer]
Hi @tarekabouzeid, to enable cost estimation for Trino, first you need to change `superset_config.py` file:
```
FEATURE_FLAGS = {
    "ESTIMATE_QUERY_COST": True
}
```
Next, specify the version of Trino that you are using in the `Advanced > Other > Version` in the `Database Connections` setup on Superset.
